modules/ROOT/pages/latest-overview-concept.adoc

```diff
@@ -1,68 +1,177 @@
 +
-* WSDL for SOAP APIs
+For details, see xref:defining-sla-tiers.adoc[Reviewing SLA Tiers Concepts].
+
+<8> The tracking registration status of each API: *Active*, *Inactive*, *Deploying*, or *Unregistered*.  
++
+A status of *Unregistered* means that Anypoint Platform has never tracked the endpoint for this API version. Either you have entered a URL for an API or proxy that is not yet deployed, or you have declared an endpoint that is hosted somewhere other than an API Gateway and need to proxy that endpoint so that the platform can track it.
 +
-Provide the URL where Anypoint Platform can find Web Services Definition Language source.
+The endpoint must have a tracking registration status of *Active* for governance policies and SLA tiers to function.
+
+<9> The number of contracted client applications for each API. For details, see xref:api-contracts-landing-page.adoc[Client Applications and Contracts].
+
+
+== API Manager Components and Concepts
+
+=== Policies
+
+Policies enable you to enforce regulations to help manage security, control traffic, and improve adaptability of your APIs. For example, a policy can control authentication, access, allotted consumption, and service level access (SLA).
+
+API Manager supports the following types of policies:
+
+* Default policies
+* Custom policies
+
+For details, see xref:policies-landing-page.adoc[Policies].
+
+=== SLA Tiers
+
+Service Level Access (SLA) tiers are categories of user access that you define for an API. The tier definition combined with an SLA-based policy determines whether access to the API at a certain level requires your approval. The tier definition also can limit the number of requests an application can make to the API. To enforce SLA tiers, you need to apply a rate-limiting or throttling policy that is SLA-based.
+
+For details, see xref:defining-sla-tiers.adoc[Reviewing SLA Tiers Concepts].
+
+=== Client Providers (Identity Providers)
 
-// talk about naming, instances
+You use client providers to enforce security and regulations in your business organization. Client providers authorize client applications. 
 
-After configuring the API, you can perform API management tasks, applying policies and setting up SLA tiers, assigning permissions to the API environment,
-versioning, viewing API analytics, and more.
+For details, see xref:configure-multiple-credential-providers.adoc[Configure Multiple Client Providers for Client Management].
 
-The API dashboard shows key information about a deployed API instance:
+=== Contracts
+
+Contracts define how client applications consume APIs. A client application requests access in Exchange. Either the owner of the API instance approves the request in API Manager, or the approval is automatic, depending on configuration. 
+
+Contracts are enforced with either of the following:
+
+* SLA enforcement policies
+* Client enforcement policies
+
+For details, see xref:api-contracts-landing-page.adoc[Client Applications and Contracts].
```

Comment:
GLOBALLY try to specify what details, specifically, we are referring the audience to: what information that that location relates directly to this topic? Details about WHAT (without restating the title of this subtopic). For example, do we mean...:
```suggestion
For details about enforcement policies, see xref:api-contracts-landing-page.adoc[Client Applications and Contracts].
```

Reply:
...the GLOBAL being to replace "for details" with something more specific, if appropriate or possible.

Reply:
Fixed here, and fixed in a few other places where it made sense. I feel it didn't make sense in situations where the title of the link was self-explanatory, or if the topic title itself described what was being linked to.